modules/expo-localization/src/Localization.web.js

```diff
@@ -0,0 +1,2 @@
+// @flow
```

Comment:
Are we still targeting `web` too even with empty files? 🤔

Reply:
This is just to prevent the module from breaking with `react-native-web`